### DISCUSSION TITLE
Project oder GitHub

### DISCUSSION THREAD

**Discussion by damboeck** [LeTTo-GmbH/LeTTo#5] [General]
Ich starte hier mal eine Diskussion ob wir Chats, Tickets etc. auf dem "project" lassen oder dafür das GitHub-System verwenden sollten.

**Comment by christiankral** [comment]
Wenig überraschend bin ich für eine Umstellung auf GitHub. Ich liste Gründe auf, die für einen Umstieg sprechen:

1. **Effiziente Ticketbarbeitung**: Ich kann in GitHub Tickets viel effizienter erstellen, etwa um `Code` zu erstellen. Das funktioniert in OpenProject theoretisch auch, praktisch aber nicht so, dass man damit zügig arbeiten könnte.
2. **Ticketreferenzierung**: Wenn ich in diesem Ticket #5 auf das Ticket auf #4 verweise wird auch in Ticket #4 automatisch auf #5 verwiesen. Ich muss nicht manuell doppelt verweisen um eine konsistente Darstellung zu erhalten. Außerdem kann man sogar auf bestimmte Replys anworten und auf Replys in anderen Tickets verweisen und damit deutlich strukturierter und effizienter arbeiten.
3. **Entwicklungsprozess**: In der jetzigen Entwicklungsphase vor der nächsten Stable-Version wäre es aus meiner Sicht sehr sinnvoll, auch Milestones zu verwenden, z. B.
    - Milestone **Stable**
    - Milestone **Bug fix**
    Damit wäre eine zusätzliche Priorisierung beim Abarbeiten von Tickets möglich.
4. **Codeverweise**: Man kann einfach auf committs von anderen Repositories verweisen. Das wäre für euren  @damboeck @l-mayt Entwicklungsprozess gut, weil ihr damit immer eine volle Verlinkung in einem einheitlichen  System habt. Beispiel siehe https://github.com/LeTTo-GmbH/LeTTo/issues/1#issuecomment-2395392160
5. Das **Benachrichtigungsystem** funktioniert unter GitHub deutlich besser als unter Projekt. Wann immer ein Änderung an einem Ticket durchgeführt wird, erhält man eine E-Mail. Das halte ist für meinen persönlichen Workflow sehr wichtig. Unter OpenProject hat das mal funktioniert, aber es funktioniert nicht mehr. Es gibt "nur" noch einmal am Tag eine Benachrichtigung oder wenn man jemanden mit @ referenziert.
6. Man kann einfache **Erledigungslisten** implementeriern:
    - [ ] Dieser Inhalt wurde von @damboeck gelesen
    - [ ] Dieser Inhalt wurde von @l-mayt gelesen
    - [ ] Dieser Inhalt wurde von @l-asch gelesen
    - [x] Dieser Inhalt wurde von @christiankral gelesen

Außerdem ist der Umstieg keine Hexerei. Die meisten User verwenden ohnehin 

- entweder nur die Headline 
- reinen Text
- Bilder (als Anhang) oder direkt im Editor

Und das alles ist vom Handling kein ernsthaftes Problem.

**Reply by christiankral** [reply]
@l-asch @l-mayt Zur Diskussion

**Comment by christiankral** [comment]
Ich habe noch ein Argument vergessen, das gegen OpenProject spricht:

![image](https://github.com/user-attachments/assets/fc1bde55-12ff-4366-a26f-ffbbb466e534)

**Comment by christiankral** [comment]
@damboeck @l-mayt  @l-asch  Ich sehe, es gibt keine Widersprüche. Das werte ich als gutes Zeichen für die Implementierung von GitHub anstelle von OpenProj.
